content/800-data-platform/400-members/100-add.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Add a team member'
+metaTitle: 'Add a team member'
+metaDescription: 'In Prisma Data Platform, you can invite team members to a project when you just define GitHub username and assign them a role.'
+---
+
+<TopBlock>
+
+In Prisma Data Platform, you can invite team members to a project when you just define GitHub username and assign them a role.
+
+</TopBlock>
+
+## Prerequisites
+
+- You are the [Project owner](/data-platform/members/roles-permissions) or have the [Admin](/data-platform/members/roles-permissions) role.
```

Comment:
I see links from the `change-role` and `remove` pages are bold, but these are not, should we make it consistent?